src/main/scala/com/ironcorelabs/SucceededResult.scala

```diff
@@ -0,0 +1,22 @@
+package com.ironcorelabs.scala.sdk
+
+import com.ironcorelabs.{sdk => jsdk}
+
+/**
+ * A successful edit of the access list of a document.
+ *
+ * @param users a list of users whose access was granted/revoked.
+ * @param groups a list of groups whose access was granted/revoked.
+ */
+case class SucceededResult(
```

Comment:
This whole structure could be replaced by `List[UserOrGroupId]` if that's of benefit?

Reply:
After considering it a bit, I think this type is useful - Makes it easier to know what was successful and what was failed.